style.css

```diff
@@ -1,3 +1,13 @@
-h1 {
-  color: red;
+* {
+  margin: 0;
+  padding: 0;
+}
+body {
+  padding: 16px;
+}
+header {
+  width: 100%;
+}
+img {
+  width: 100%;
```

Comment:
Looks like the prettier extension is not working correctly, since there should be an empty line after each declaration.